app/models/project_mediaflux.rb

```diff
@@ -33,16 +33,18 @@ def self.create!(project:, session_id:, xml_namespace: nil)
           #All exceptions that are raised should include the current expected metadata schema version number, as well as what metadata is missing.
     id = create_request.id
     if id.blank?
-      response_xml = create_request.response_xml
-      response_text = response_xml.text
-      case response_text
+      response_error = create_request.response_error
+      case response_error[:message]
       when "failed: The namespace #{project_namespace} already contains an asset named '#{project_name}'"
         raise "Project name already taken"
       when /'asset.create' failed/
 
         # Ensure that the metadata validations are run
-        project.metadata_model.validate
-        raise TigerData::MissingMetadata.missing_metadata(schema_version:"0.6", errors: project.metadata_model.errors)
+        if project.valid? &&  project.metadata_model.valid?
+          raise response_error[:message]  # something strange went wrong
+        else
+          raise TigerData::MissingMetadata.missing_metadata(schema_version:"0.6", errors: project.metadata_model.errors)
```

Comment:
Should this use `TigerdataSchema::SCHEMA_VERSION` instead of hard coding the version number?

Reply:
Likely.  That was not the code change I was working on, but I got tired of not seeing the real error from Mediaflux.  I'll update it.